client/src/components/aggregations/utils.tsx

```diff
@@ -241,27 +218,22 @@ export function reportsToEvents(reports, showInterlocutors) {
         // and finally ascending by title
         r1.title.localeCompare(r2.title)
     )
+
+  // Do we have an event to show as well?
+  if (event) {
+    result.push(createCalendarEventFromEvent(event))
+  }
+
+  return result
 }
 
-export function eventsToCalendarEvents(events) {
+export function eventsToCalendarEvents(events, showInterlocutors) {
+  // Show in the calendar all events and all its reports
   return events
-    .map(event => {
-      let title = `${event.name}`
-      if (event.location) {
-        title = `${title}@${event.location.name}`
-      }
-      const start = new Date(event.startDate)
-      start.setSeconds(0, 0) // truncate at the minute part
-      const end = new Date(event.endDate)
-      end.setSeconds(0, 0) // truncate at the minute part
-      return {
-        title,
-        start,
-        end,
-        url: Event.pathFor(event),
-        extendedProps: { ...event },
-        allDay: !Settings.eventsIncludeStartAndEndTime
-      }
+    .flatMap(event => {
+      return [createCalendarEventFromEvent(event)].concat(
+        reportsToEvents(event.reports, showInterlocutors, null)
```

Comment:
This is odd: you fetch a sorted list of calendar events for report, then below sort them again, but with the sorting strategy for events.